ci/deps/actions-310.yaml

```diff
@@ -22,36 +22,36 @@ dependencies:
   - pytz
 
   # optional dependencies
-  - beautifulsoup4
+  - beautifulsoup4>=4.9.3
   - blosc
-  - bottleneck
-  - brotlipy
-  - fastparquet
-  - fsspec
-  - html5lib
-  - hypothesis
-  - gcsfs
-  - jinja2
-  - lxml
+  - bottleneck>=1.3.2
+  - brotlipy>=0.7.0
+  - fastparquet>=0.6.3
+  - fsspec>=2021.07.0
+  - html5lib>=1.1
+  - hypothesis>=6.34.2
+  - gcsfs>=2021.07.0
+  - jinja2>=3.0.0
+  - lxml>=4.6.3
   - matplotlib>=3.6.1, <3.7.0
-  - numba
-  - numexpr
-  - openpyxl<3.1.1
-  - odfpy
-  - pandas-gbq
-  - psycopg2
-  - pymysql
-  - pytables
-  - pyarrow
-  - pyreadstat
-  - python-snappy
-  - pyxlsb
+  - numba>=0.53.1
+  - numexpr>=2.7.3
+  - openpyxl>=3.0.7, <3.1.1
+  - odfpy>=1.4.1
+  - pandas-gbq>=0.15.0
+  - psycopg2>=2.8.6
+  - pymysql>=1.0.2
+  - pytables>=3.6.1
+  - pyarrow>=7.0.0
+  - pyreadstat>=1.1.2
+  - python-snappy>=0.6.0
+  - pyxlsb>=1.0.8
   - s3fs>=2021.08.0
-  - scipy
-  - sqlalchemy
-  - tabulate
-  - tzdata>=2022a
```

Comment:
this one's really odd, looks like need to keep it as `2022a` if it's in conda, and `2022.1` if it's in pip

Reply:
Oh, my mistake. For some reason I made the assumption that they were interchangeable. What should be done about it? I see `EXCLUDE_DEPS` contains `tzdata`. Should this be added into the exclusion as well, or is there a way to determine if it is in pip or conda?

Reply:
It's probably fine to exclude it - you could do some complex code to handle it, but I'm not sure it'd be worth the effort to be honest